### ISSUE TITLE
Publication bias

### ISSUE THREAD

**Issue by karafecho** [NCATSTranslator/Feedback#77]
The QotM surfaced a (known) issue related to publication bias. Specifically, Translator relies heavily on scientific publications for reasoning and assertions, which presents challenges when studying a novel compound with essentially no relevant publications, e.g., SRI-37330. Not unique to Translator, this challenge question revealed that while it is relatively easy to find small molecules that bind to a known target, it is not easy to find a molecular target for a novel small molecule, even if the function of that small molecule has been established.

**Comment by andrewsu** [comment]
My diagnosis of the SRI-37330 example ([related issue](https://github.com/NCATSTranslator/Feedback/issues/67)) is a little different.  The issue with Translator not returning data for SRI-37330 (and other less-well-studied compounds) is I think not so much the fact that Translator biases toward sources with publication evidence, but the lack of _structured_ data resources that we can ingest with relevant data.  If you do a simple [web search for "SRI-37330"](https://www.google.com/search?q=SRI-37330), the top hits are chemical vendors, which generally do not expose their data in structured formats.  Neither ChEMBL nor ChEBI have entries for this compound (when searching by "SRI-37330", the CAS number "2322245-49-6", or the InchiKey "SIWGSSRKYJTJTF-UHFFFAOYSA-N").  PubChem does have [an entry for this compound](https://pubchem.ncbi.nlm.nih.gov/compound/142582737), but it does not capture any relationships to other entities.  And that makes sense to me since a [PubMed search](https://pubmed.ncbi.nlm.nih.gov/?term=SRI-37330) for this compound currently only returns a single relatively-recent paper.  

TLDR: I don't see any problem with Translator (at least not one that lends itself to a specific change that we can make), so I'd propose closing this issue.

**Comment by karafecho** [comment]
For the SRI-37330 example, I completely agree with your thoughts. However, upon revisiting my post, I think I conflated two issues: (1) the SRI-37330 example and (2) a broader issue related to a bias toward recent publications on 'hot' topics or heavily researched topics, e.g., COVID, cancer. (1) was addressed by MolePro under #67. (2) is related to #79. 

Posted additional notes under 79, so closing this issue ...
